ingest/config/defaults.yaml

```diff
@@ -16,10 +16,8 @@ oc43:
 hku1:
   ncbi_taxon_id: "290028"
   annotations: "config/hku1/annotations.tsv"
-
 # Optional fields to add to the NCBI Datasets output
 ncbi_dataset_fields: []
-
 # Config parameters related to the curate pipeline, shared by all viruses
 curate:
```

Comment:
Generally I prefer keeping the newlines here - it visually pairs comments with the line directly under it. We use this pattern across various repos and in different types of files.

If the precommit config can be tweaked to allow these types of newlines, that'd be great. From a quick glance at [yamlfmt config options](https://github.com/google/yamlfmt/blob/629504c4fa2d39c880a27a6c9cf1cf1bbc3bfae0/docs/config-file.md), I think this can be done by setting `retain_line_breaks` or `retain_line_breaks_single` to `true`.

Reply:
Oh I missed the `.yamlfmt` file in my review - good to know we're on the same page!

Reply:
pushed a fix for both config files restoring the previously-removed lines